organization/mission.md

```diff
@@ -22,9 +22,10 @@ The founding principles were adopted by the board of directors on 23 August 2019
 
 \* In order to start as an association 4 initial individual founders will play the role of member until there are enough public members for the organization to sustain itself.
 
-## Goals
+## Current priorities
 
-* Enable an **ecosystem of Public Code software products**, developed by and for public administrations to solve real problems for citizens
-* Build a **collaborative community of developers and cities**, which continuously improves and maintains Public Code products
-* Provide a **certification system for software** as meeting the standard set by the Foundation For Public Code
-* Build a **trusted brand and an awareness of the value of Public Code**
+The current priorities of the organization are decided on by the [board of directors](governance-model.md). They are:
+
+* Members: get public organizations to join the association
+* Codebases: get codebases into stewardship
+* Funding: secure funding for the next years
```

Comment:
I suggest you add a date to these priorities, like with the founding principles.
Suggested sentence:
These priorities were adopted by the board of directors on [date].

Reply:
I've done this in the commit message, as I think this is more important as historical documentation. That way we can document the changes without extra noise.

I'm curious as to what you think about that. I personally think this is something we should use Git more for, but understand that some of this might also be about the flat value of it.

Reply:
Like linking to a 'past milestones' page, adding a date makes it explicitly clear that our priorities aren't static - and that ideally, they evolve based on what we accomplish. Putting the date into a commit message makes it quite hard to find in case anyone were curious.

Reply:
That first argument I think is a good one, for the second one, I wonder whether that edge case weighs up against having two separate version control mechanisms and the possibility of making more mistakes in updating the content. I'll go think about a good way to do this whilst not incurring a lot more overhead.